prow/cmd/deck/main.go

```diff
@@ -195,6 +199,16 @@ func gatherOptions(fs *flag.FlagSet, args ...string) options {
 	o.storage.AddFlags(fs)
 	o.pluginsConfig.AddFlags(fs)
 	fs.Parse(args)
+
+	if o.spyglassFilesLocation == "" {
+		o.spyglassFilesLocation = fmt.Sprintf("%s%s", os.Getenv("KO_DATA_PATH"), defaultSpyglassFilesLocation)
```

Comment:
Nit: These defaults can still be defined inline with the flag definitions.

Reply:
good point, doen